tests/foreman/rhai/test_rhai.py

```diff
@@ -101,3 +102,35 @@ def test_org_selection_for_rhai(self):
             result = session.nav.wait_until_element(
                 locators['insights.org_selection_msg']).text
             self.assertIn("Organization Selection Required", result)
+
+    def test_unregister_system_from_rhai(self):
+        """@Test: Verify that 'Unregister' a system from RHAI works correctly
+
+        @Feature: If a machine if unregistered from the RHAI web interface,
+        then the client should be able to use the service.
+
+        @Assert: Once the machine is unregistered from the RHAI web interface
+        then the unregistered client machine should return a 1 on running the
+        service 'redhat-access-insights'
+
+        """
+        # Register a VM to Access Insights Service
+        self.rhai.register_client_to_rhai(
+            self.ak_name,
+            self.org_label,
+        )
+        with Session(self.browser) as session:
+            session.nav.go_to_select_org(self.org_name)
+            Navigator(self.browser).go_to_insights_systems()
+            # Click on the unregister icon 'X' in the table against the
+            # registered system listed.
+            session.nav.wait_until_element(
```

Comment:
Please, use click() method according to https://github.com/SatelliteQE/robottelo/pull/2556